uavcan/src/transport/can/mod.rs

```diff
@@ -236,9 +230,7 @@ impl<'a, C: Clock> Iterator for CanIter<'a, C> {
             self.payload_offset += bytes_left;
             unsafe {
                 frame.payload.push_unchecked(
-                    TailByte::new(true, true, true, self.transfer.transfer_id)
-                        .to_u8()
-                        .unwrap(),
+                    TailByte::new(true, true, true, self.transfer.transfer_id).0
```

Comment:
Add a function to retrieve the byte (u8).

Reply:
Will address in another issue/PR